packages/web-pixel-ui-extensions/src/extension-points/web-pixel-api.ts

```diff
@@ -0,0 +1,23 @@
+export type Version = 'unstable';
```

Comment:
Why won't this package be the source of those types?

Reply:
I'd like to keep the types within the pixel manager repo and have this package pull the latest types. Unless you have another strategy that you'd like to share, I'm all ears.

Reply:
In checkout, we make the package in this repo be the source of truth, and we depend on it in our private repos that care about types it contains. I like this model better because it means the "real" value is something that both we and extensions can see, and it prevents any weird setups where consumers have to look at multiple repos to understand types or, worse yet, where some subset of the types they care about are in private repos.

Reply:
Sounds good, we're on board to have this package be the source of truth for types. Our hesitation was based on the fact that we generate schemas and their types for standard events and we weren't clear on wether we wanted that generation tool to live here. Seems like the best choice to have the schema definition here as it will serve as documentation for partners.